### DISCUSSION TITLE
How can I contribute?

### DISCUSSION THREAD

**Discussion by joaolucasmacedo** [becarpenter/book6#14] [Show and tell]
Hi guys. I have book draft on IPv6. It has more than 100 A4 pages, but I did not finished the job. 
I have written most of the fundamental/theoretical parts, but it still lacks some "hands on" material, although I have written some command line references.
It is in Brazilian Portuguese, but I can translate each article and submit here to your evaluation.

**Comment by becarpenter** [comment]
That sounds great! Can you review our draft table of contents and comment on how your work could fit in there? Nothing is frozen yet. Also review CONTRIBUTING.md - again, comments and suggestions are very welcome.

**Comment by becarpenter** [comment]
@joaolucasmacedo, any progress? We still have multiple sections in need of text...

**Reply by joaolucasmacedo** [reply]
Hi Brian. Thanks for pinging me. I've seen your progress here.
I'm translating the topic on **Network Desing/Address Planning**. You can see on my fork.
I've been kinda busy theses days, but I'll try to help.

**Comment by becarpenter** [comment]
That's great. Let me know when it is ready for review. I will be happy to edit it for syntax/grammar issues.

I expect there will be many opinions on this topic once some text is published on the main branch.
